setup.py

```diff
@@ -2,8 +2,11 @@
 
 setup(
     name='erezutils',
-    version='1.0.0',
+    version='1.1.0',
```

Comment:
I'd prefer to bump the version in a separate commit to keep ti separate from the feature.